docs/angular-integration.asciidoc

```diff
@@ -6,7 +6,7 @@ This document covers how to use Real User Monitoring JavaScript agent with Angul
 [[angular-supported-versions]]
 ==== Supported versions
 
-This integration supports Angular versions < 9.
+This integration supports Angular versions ≥ 9.0.
```

Comment:
Technically the [peerDependencies](https://github.com/elastic/apm-agent-rum-js/pull/1001/files#diff-3482bae83fe598778f668593dc4d3f853d0bb0a66aa9039a3dc4eba384cd1502R37-R38) list supported version as `^9.0.0`. While it may work NPM will complain about unmet dependencies if using Angular 10,11 or 12

Reply:
It should be just a warning, should not break as such, I do think with npm 7 it going to automatically install the peer deps, have to test it though.

Reply:
Right, but what I'm saying is if you're supporting multiple Angular versions then specify it so NPM won't complain. There will be subsequent issues filed from devs using this because of the warning. Specifically like `>=9.0.0 <12.0.0`

Reply:
Gotcha 👍